### PR TITLE
Run Nginx & Django throught https

### PR DESCRIPTION
**Pull request by Neffi42**

### CODE REVIEW

File: .env.template [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Nginx
 HOSTNAME=
-BASE_URL=
+BASE_URL= # https//${HOSTNAME}.com
```
Comment (antoineverin) [inline .env.template:3]: Missing `:` in https://

File: django/conf/certs.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+set -e
+
+mkdir -p /static /etc/ssl/private /etc/ssl/certs
+
+openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /etc/ssl/private/${HOSTNAME}.key -out /etc/ssl/certs/${HOSTNAME}.crt -subj "/CN=${HOSTNAME}"
+
+chmod 644 /etc/ssl/private/${HOSTNAME}.key /etc/ssl/certs/${HOSTNAME}.crt
```
Comment (antoineverin) [inline django/conf/certs.sh:8]: Could be both on the same directory?

File: docker-compose.yml [hunk 3]
```diff
@@ -35,14 +38,16 @@ services:
...
-      - 8080:80
+      - 4443:443
     init: true
     networks:
       - network
     volumes:
       - nginx-src:/usr/share/nginx
       - static-files:/static
       - media-files:/media
+      - public-keys:/etc/ssl/certs/
```
Comment (antoineverin) [inline docker-compose.yml:49]: Could use one volume with the parent directory

**Review by antoineverin — CHANGES_REQUESTED** [review]

File: .env.template (hunk 1 above)
Comment (Neffi42) [reply .env.template:3]: oops

File: django/conf/certs.sh (hunk 2 above)
Comment (Neffi42) [reply django/conf/certs.sh:8]: they could for sure

**Review by antoineverin — APPROVED** [review]

**Pull request merged**